### DISCUSSION TITLE
NodeJS full tutorial

### DISCUSSION THREAD

**Discussion by AugustinMauroy** [mdn/mdn-community#522] [Content] [answered]
# NodeJS full tutorial
Hello everyone, I create this discussion to know if people would be able to maintain the tutorial on Node.JS.

I have started the work see this branch of my fork [here](https://github.com/AugustinMauroy/content/tree/Learn-nodeJS-A-to-Z).

Being a French speaker I can do the translation of these documents.

**Comment by MrBrain295** [comment, marked as answer]
Discussions have moved to [here](https://github.com/mdn/mdn-community/discussions).

**Reply by AugustinMauroy** [reply]
its done
